contracts/0.8.9/LidoLocator.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+
+// SPDX-License-Identifier: GPL-3.0
+
+/* See contracts/COMPILERS.md */
+pragma solidity 0.8.9;
+
+import {ILidoLocator} from "../common/interfaces/ILidoLocator.sol";
+
+/**
+ * @title LidoLocator
+ * @author mymphe
+ * @notice Service Locator of Lido
+ */
+contract LidoLocator is ILidoLocator {
+    error ErrorIncorrectLength();
+    error ErrorZeroAddress();
+
+    address internal immutable lido;
+    address internal immutable compositePostRebaseBeaconReceiver;
```

Comment:
maybe `UPPER_CASE_WITH_UNDERLINES` here and there?

Reply:
I'd rather stick with public immutables but in camel case.

Reply:
And without getters.